requirements/requirements.in

```diff
@@ -39,6 +39,7 @@ django_mptt
 django_reversion
 django_taggit
 djangorestframework
+drf-extensions
```

Comment:
A change to `requirements.in` resulted in `pip-compile` modifying `external_services.txt`? I think I need to go back and read https://github.com/kobotoolbox/kpi/pull/1148.